src/CssVariables.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import * as theme from './theme'
```

Comment:
This is a contentious topic, but we have standardised on being explicit about the file extension. I don't care now, but to follow the conventions in the project it is good to be aware of.

Reply:
Ah I didn't catch that. Wait, I'll add it.

Reply:
Done